modules/tsb/mp/manifests/tsb/managementplane-values.yaml.tmpl

```diff
@@ -35,7 +35,7 @@ spec:
       issuers:
         - name: https://jwt.tetrate.io
           signingKey: private.key
-      signingKeysSecret: iam-signing-key
+      signingKeysSecret: custom-iam-signing-key
```

Comment:
Probably better to remove the issuers list to use the defaults? Less internal details exposed as config.